PumaRhino/ModelGenerator.cpp

```diff
@@ -396,8 +396,8 @@ void ModelGenerator::extractMainShapeAttributes(pcu::AttributeMapBuilderPtr& aBu
 		if (convertShapeAttr->hasKey(L"startRule") &&
 		    convertShapeAttr->getType(L"startRule") == prt::AttributeMap::PT_STRING)
 			startRule = convertShapeAttr->getString(L"startRule");
-		if (convertShapeAttr->hasKey(SEED_KEY) && convertShapeAttr->getType(SEED_KEY) == prt::AttributeMap::PT_INT)
-			seed = convertShapeAttr->getInt(SEED_KEY);
+		if (convertShapeAttr->hasKey(SEED_KEY) && convertShapeAttr->getType(SEED_KEY) == prt::AttributeMap::PT_FLOAT)
```

Comment:
This "fix" uncovered unclean handling of the seed parameter (or integer parameters in general).

We transport integer parameters as Double, this is not good. Please introduce integer handling in the wrapper from GH to Rhino native and PRT calls.

And for the seed value specifically, please ensure that it is transported as int32_t on the native side. We currently have a somewhat problematic chain of GH_Integer (Grasshopper) -> Double (wrapper) -> int (ShapeAttribute helper class) -> int32_t (PRT API)